lib/importmap/packager.rb

```diff
@@ -39,16 +39,18 @@ def pin_for(package, url)
   def vendored_pin_for(package, url)
     filename = package_filename(package)
     version  = extract_package_version_from(url)
-
-    if "#{package}.js" == filename
-      %(pin "#{package}" # #{version})
-    else
-      %(pin "#{package}", to: "#{filename}" # #{version})
-    end
+    line_formatted_pin_options = pin_options_for_package(package).except("to").map { |option, value| %(#{option}: #{value.is_a?(String) ? %("#{value}") : value}) }
```

Comment:
An alternative to managing the `to` is to update the pin options hash `to` with the new value instead of straight up excluding it first and then reintroducing it again in `pin_components`.

The one benefit to the alternative is that it maintains the original order of the pin options. This makes it a little nicer to review changes on git (and Github) since the ordering is maintain and thus highlighted nicely.

Reply:
Changes made to preserve ordering, https://github.com/rails/importmap-rails/pull/274/commits/1fca7d78b14cfa409bf3e6eabbba97675aade929